### ISSUE TITLE
Reg\Dashboard - External Users can see dashboard tiles

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1609]
#### Description:
As an external user, I have a registration dashboard, so that I can conduct registration related activities
[External user Figma link](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=7702-4102&mode=design&t=KFDAnKMzwJ3b1l14-4)

For the styling (font, colour, margin, padding, etc.), please refer to [this frame on the BCIERS Design Guidelines page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=11103-107&t=pmKjK1RmQeLYOb04-4) in Figma.

#### Acceptance Criteria:
Given I'm an industry user and I have access to the Registration tile on the BCIERS dashboard
When I click the Registration link in the Registration tile
Then I see the External - Registration page as in the [design](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7702-4102&t=pmKjK1RmQeLYOb04-4)
When I click on the sub links of the Registration tile
Then I'm navigated to the proper page as in the [prototype](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?page-id=7692%3A2&type=design&node-id=7702-4102&viewport=153%2C235%2C0.04&t=9CHcuF7tZEcxkUP7-1&scaling=scale-down&starting-point-node-id=7692%3A3728&show-proto-sidebar=1)

#### Development Checklist:
- [ ] in `bciers/apps/registration/app/dashboard/page.tsx` get tile JSON data using `fetchDashboardData from "@bciers/actions/server";` with parameter `"common/dashboard-data?dashboard=registration"`
- [ ] send the tile JSON data to `Tiles from "@bciers/components/navigation/Tiles";`
- [x] make Tiles pretty

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by #1617

**Comment by nanyangpro** [comment]
Hi @hannavovk - This ticket has been updated with the proper links and AC. For any developers who may have questions, happy to hop on a call to fine tune the AC together.

**Comment by shon-button** [comment]
@patriciarussellCAS 

**Reachable at:**
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/registration

**Comment by patriciarussellCAS** [comment]
Thanks @shon-button - this looks good. Though I see that the Registration dashboard in Dev is outdated (Register an Operation, and Report an Event) but should now includes 3 tiles: 

- Register an Operation
- Closures, Temporary Shutdowns, Restart Operations
- Transfer an Operation or Facility

If it works for you, I will close this ticket and create a follow-up ticket for that work. 
Tagging: @marcellmueller for visibility
